### ISSUE TITLE
Error in voot

### ISSUE THREAD

**Issue by lilprofcragz** [biezom/hotstarsportslivestreamer#6]
D:\Tools\vootstreamer>vootlivestreamer.bat
paste the link:http://www.voot.com/kids/characters/chhota-bheem-h-/368477?gclid=CjwKEAjw7e66BRDhnrizmcGc8VcSJABR6gaRSfaAiBldhUqdyN26RRW-nkB3u7fM3frmImexwsznAhoCoBjw_wcB

Notice: Undefined offset: 1 in D:\Tools\vootstreamer\voot.php on line 21

Notice: Undefined offset: 1 in D:\Tools\vootstreamer\voot.php on line 23

Notice: Undefined offset: 1 in D:\Tools\vootstreamer\voot.php on line 27

Notice: Undefined offset: 1 in D:\Tools\vootstreamer\voot.php on line 27

Notice: Undefined offset: 1 in D:\Tools\vootstreamer\voot.php on line 27
write bitrate:1080p

Notice: Undefined offset: 1 in D:\Tools\vootstreamer\voot.php on line 21

Notice: Undefined offset: 1 in D:\Tools\vootstreamer\voot.php on line 23

Notice: Undefined offset: 1 in D:\Tools\vootstreamer\voot.php on line 27

Notice: Undefined offset: 1 in D:\Tools\vootstreamer\voot.php on line 27

Notice: Undefined offset: 1 in D:\Tools\vootstreamer\voot.php on line 27

Notice: Undefined offset: 1 in D:\Tools\vootstreamer\voot.php on line 50

Notice: Undefined offset: 1 in D:\Tools\vootstreamer\voot.php on line 50
http://video.voot.com/enc/fhls/p/1982551/sp/198255100/serveFlavor/entryId//v/2/pv/1/flavorId//name/a.mp4/index.m3u8

Starting  livestreamer...

[cli][info] Found matching plugin stream for URL hls://http://video.voot.com/enc/fhls/p/1982551/sp/198255100/serveFlavor/entryId//v/2/pv/1/flavorId//name/a.mp4/index.m3u8
[cli][info] Available streams: live (worst, best)
[cli][info] Opening stream: live (hls)
[stream.hls][debug] Reloading playlist
[cli][error] Could not open stream: Unable to open URL: http://video.voot.com/enc/fhls/p/1982551/sp/198255100/serveFlavor/entryId//v/2/pv/1/flavorId//name/a.mp4/index.m3u8 (404 Client Error: Not Found)
Done.
Press any key to continue . . .

I am getting this error message while trying to download from voot.
I have downloaded and extracted the latest version from github today.

**Comment by s841989** [comment]
same problem

**Comment by imlokesh** [comment]
Enter your link like this:
`http://www.voot.com/kids/characters/chhota-bheem-h-/368477`
Remove everything after questionmark.

**Comment by s841989** [comment]
no, it's not working. still displaying undefined offset 1 error

**Comment by biezom** [comment]
hi
link is not complet
example
<pre>http://www.voot.com/kids/characters/chhota-bheem-h-/368477/the-greatest-archer-of-dholakpur/374095</pre>

**Comment by s841989** [comment]
link is complete. but it shows undefined offset in line 21

**Comment by neerajthenext** [comment]
same problem here

**Comment by s841989** [comment]
hi, I found the reason for undefined offset. Livestreamer is not working on proxy adress. but when connecting to mobile hotspot, livestreamer is working on sim internet. But its not working on Lan connection. Now tell the solution, when will it work on Lan connection.

**Comment by s841989** [comment]
paste the link:http://www.voot.com/shows/comedy-nights-bachao/1/360658/when-gags
waaley-met-dilwale/358059
write quality (write low medium or high):low

Notice: Undefined offset: 1 in C:\hotstarsportslivestreamer-master\voot.php on l
ine 22

Notice: Undefined offset: 1 in C:\hotstarsportslivestreamer-master\voot.php on l
ine 24

Notice: Undefined offset: 1 in C:\hotstarsportslivestreamer-master\voot.php on l
ine 25

Notice: Undefined offset: 1 in C:\hotstarsportslivestreamer-master\voot.php on l
ine 26

Notice: Undefined offset: 1 in C:\hotstarsportslivestreamer-master\voot.php on l
ine 28

Notice: Undefined offset: 1 in C:\hotstarsportslivestreamer-master\voot.php on l
ine 29

Notice: Undefined offset: 1 in C:\hotstarsportslivestreamer-master\voot.php on l
ine 31

Notice: Undefined offset: 1 in C:\hotstarsportslivestreamer-master\voot.php on l
ine 32

Notice: Undefined offset: 1 in C:\hotstarsportslivestreamer-master\voot.php on l
ine 35

Notice: Undefined offset: 1 in C:\hotstarsportslivestreamer-master\voot.php on l
ine 46
http://video.voot.com/enc/fhls/p/1982551/sp/198255100/serveFlavor/entryId//v/2/p
v/1/flavorId//name/a.mp4/index.m3u8

Starting  livestreamer...

[cli][info] Found matching plugin stream for URL hls://http://video.voot.com/enc
/fhls/p/1982551/sp/198255100/serveFlavor/entryId//v/2/pv/1/flavorId//name/a.mp4/
index.m3u8
[cli][info] Available streams: live (worst, best)
[cli][info] Opening stream: live (hls)
[cli][error] Could not open stream: Unable to open URL: http://video.voot.com/en
c/fhls/p/1982551/sp/198255100/serveFlavor/entryId//v/2/pv/1/flavorId//name/a.mp4
/index.m3u8 (404 Client Error: Not Found)
Done.
Press any key to continue . . .
.
I am getting this error message while trying to download from voot. Plz tell the solution

**Comment by batistatista** [comment]
plz give the relevant solution for this facing same problem (404 Client Error: Not Found)

**Comment by batistatista** [comment]
or is there any anroid app through which we can download??????

**Comment by ks348** [comment]
Starting livestreamer...

[cli][info] Found matching plugin stream for URL hls://http://video.voot.com/enc
/fhls/p/1982551/sp/198255100/serveFlavor/entryId//v/2/pv/1/flavorId//name/a.mp4/
index.m3u8
[cli][info] Available streams: live (worst, best)
[cli][info] Opening stream: live (hls)
[cli][error] Could not open stream: Unable to open URL: http://video.voot.com/en
c/fhls/p/1982551/sp/198255100/serveFlavor/entryId//v/2/pv/1/flavorId//name/a.mp4
/index.m3u8 (404 Client Error: Not Found)
Done.
Press any key to continue . . .
.
I am getting this error message from 4 days while trying to download from voot.com Plz tell the solution

**Comment by Mki-vrm** [comment]
![error](https://cloud.githubusercontent.com/assets/14196037/21472227/c09490ae-caf4-11e6-9fef-eaf3a5e4c245.PNG)


Please resolve this. I am getting this error from last 5-6 days

**Comment by Abhishek01987** [comment]
Facing the same problem could not open stream,please solve this...

**Comment by ERHenish** [comment]
https://www.youtube.com/watch?v=l2xBDd_BgbI  check out this for 404 error in vootlive streamer

**Comment by nikhil-1337** [comment]
Getting error on ubuntu 17.0
`
bash vootffmpeg.sh
paste the link
https://www.voot.com/movie/asoka/383727
"width":128,"height":72,"bitrate":77
"width":640,"height":360,"bitrate":325
"width":640,"height":360,"bitrate":469
"width":640,"height":360,"bitrate":669
"width":640,"height":360,"bitrate":964
"width":1280,"height":720,"bitrate":1628
"width":1920,"height":1080,"bitrate":2628
"width":1920,"height":1080,"bitrate":8192
write bitrate
1628
http://vootvideo.akamaized.net/enc/fhls/p/1982551/sp/198255100/serveFlavor/entryId/0_67dn6lcz/v/2/pv/1/flavorId/0_pc3fdrhz/name/a.mp4/index.m3u8

Starting  livestreamer...

Starting  ffmpeg...

sh: 1: ffmpeg: not found
Done.
`
